packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -404,6 +407,7 @@ export const useDamFileUpload = (options: UploadDamFileOptions): FileUploadApi =
                         file,
                         folderId: targetFolderId,
                         scope,
+                        // deprecated: add importSource directly to file props instead
```

Comment:
```suggestion
```

Devs will never see this.

Reply:
19b4f456fea978097c56f6f85936b41677de46c2